src/runtime_src/core/common/api/xrt_kernel.cpp

```diff
@@ -1217,6 +1217,24 @@ class run_impl
     }
   };
 
+  //Soft Kernel Arg setter
+  struct sk_arg_setter : arg_setter
```

Comment:
Does not appear to be used.  Please complete this suggested change.

Reply:
Added more info & code.